lib/methods.js

```diff
@@ -55,6 +55,27 @@ if (Meteor.isServer) {
                     someError = true;
                 }
             });
+
+            if (AccountsTemplates.options.showReCaptcha) {
+                var secretKey = null;
+
+                if (Meteor.settings.reCaptcha && Meteor.settings.reCaptcha.secretKey) {
```

Comment:
This doesn't seem able to pickup the `secretKey` in case you configured it with:

```javascript
AccountsTemplates.configure({
    reCaptcha: {
        secretKey: YOUR SECRET KEY.
    },
});
```